docs/dev/reference/config.md

```diff
@@ -105,12 +105,11 @@ contao:
                 # Allows to convert one image format to another or to provide additional image formats for an image (e.g. WebP).
                 formats:
 
-                    # Examples:
-                    jpg:
-                        - webp
-                        - jpg
-                    gif:
-                        - png
+                    # Examples (for webp settings analogous to the options in the backend):
+                    png: [webp, png]
+                    jpg: [webp, jpg]
+                    jpeg: [webp, jpeg]
+                    gif: [webp, gif]
```

Comment:
This will probably be reverted in the future, as this part of the documentation is just the output from `config:dump-reference`.

Reply:
where can/should we add this as example?

Reply:
Directly in the configuration of the `contao/core-bundle`. However, within the documentation it would make sense to describe this part separtely. imho we should introduce a new article called "Manage image sizes" in https://docs.contao.org/manual/en/layout/theme-manager/, explain the image sizes options of the back end there and then also mention the possibility to configure image sizes in the `config.yml` directly, plus the examples.

A lot of work for sure, but having that would be worth it :)

Reply:
Possibly with links to (or text excerpts from):

- https://blog.kulturbanause.de/2014/09/responsive-images-srcset-sizes-adaptive/
- https://rocksolidthemes.com/de/contao/blog/responsive-images-picture-contao